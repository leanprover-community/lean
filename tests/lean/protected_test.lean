namespace nat
  #check induction_on      -- ERROR
  #check rec_on            -- ERROR
  namespace le
    #check rec_on          -- ERROR
  end le
end nat
