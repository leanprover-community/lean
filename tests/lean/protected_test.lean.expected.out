protected_test.lean:2:9: error: unknown identifier 'induction_on'
protected_test.lean:3:9: error: unknown identifier 'rec_on'
less_than_or_equal.rec_on :
  ?M_1.less_than_or_equal ?M_3 →
  ?M_2 ?M_1 → (∀ {b : ℕ}, ?M_1.less_than_or_equal b → ?M_2 b → ?M_2 b.succ) → ?M_2 ?M_3
less_than_or_equal.rec_on :
  ?M_1.less_than_or_equal ?M_3 →
  ?M_2 ?M_1 → (∀ {b : ℕ}, ?M_1.less_than_or_equal b → ?M_2 b → ?M_2 b.succ) → ?M_2 ?M_3
protected_test.lean:7:11: error: unknown identifier 'rec_on'
less_than_or_equal.rec_on :
  ?M_1.less_than_or_equal ?M_3 →
  ?M_2 ?M_1 → (∀ {b : ℕ}, ?M_1.less_than_or_equal b → ?M_2 b → ?M_2 b.succ) → ?M_2 ?M_3
